software/src/modules/ac011k/ac011k.cpp

```diff
@@ -291,6 +291,10 @@ void AC011K::Serial2write(byte *data, int size) {
 }
 
 void AC011K::sendCommand(byte *data, int datasize, byte sendSequenceNumber) {
+    sendCommand(ac011k_hardware.config.get("verbose_communication")->asBool(), data, datasize, sendSequenceNumber);
+}
+
+void AC011K::sendCommand(bool verbose, byte *data, int datasize, byte sendSequenceNumber) {
```

Comment:
Why not void AC011K::sendCommand(byte *data, int datasize, byte sendSequenceNumber, bool verbose = false)